site/content/docs/4.6/getting-started/accessibility.md

```diff
@@ -24,12 +24,12 @@ Because Boosted's components are purposely designed to be fairly generic, author
 
 ### Color contrast
 
-<!-- Boosted mod -->
-Some colors that currently make up [Boosted's default palette]({{< docsref "/about/brand#color-palette" >}}) —used throughout the framework for things such as button variations, alert variations, form validation indicators— may lead to *insufficient* color contrast.
+Some combinations of colors that currently make up [Boosted's default palette]({{< docsref "/about/brand#color-palette" >}})—used throughout the framework for things such as button variations, alert variations, form validation indicators—may lead to *insufficient* color contrast (below the recommended [WCAG 2.1 text color contrast ratio of 4.5:1](https://www.w3.org/TR/WCAG/#contrast-minimum) and the [WCAG 2.1 non-text color contrast ratio of 3:1](https://www.w3.org/TR/WCAG/#non-text-contrast)), particularly when used against a light background. Authors are encouraged to test their specific uses of color and, where necessary, manually modify/extend these default colors to ensure adequate color contrast ratios.
```

Comment:
I've made some modifications in this file in order to be closer to what's explained in Bootstrap while keeping the spirit of those sections. Please check if this content stays OK.

Reply:
The content is good for me. Just a typo question: is that `indicators—may...` or `indicators— may...` ?

Reply:
`indicators-may`